core/CTestScript.cmake.in

```diff
@@ -20,8 +20,5 @@ set(CTEST_SOURCE_DIRECTORY @CMAKE_SOURCE_DIR@)
 set(CTEST_BINARY_DIRECTORY @CMAKE_BINARY_DIR@)
 
 ctest_start("Continuous")
-ctest_test(EXCLUDE_LABEL broken
-           PARALLEL_LEVEL 5
-           @REPEAT_UNTIL@
-           )
+ctest_test(EXCLUDE_LABEL broken PARALLEL_LEVEL 5)
```

Comment:
I readded that since it was mostly for testing failing tests & we have some tests that fail because of missing resources where a restart can make sense.